tmt/steps/scripts/tmt-reboot

```diff
@@ -17,29 +17,32 @@ PATH=/sbin:/usr/sbin:$PATH
 
 command=""
 timeout=""
-efi=True
+efi=False
 while getopts "c:t:e" flag; do
     case "${flag}" in
         c) command="${OPTARG}";;
         t) timeout="${OPTARG}";;
-        e) efi=False;;
+        e) efi=True;;
```

Comment:
This seems odd, but I don't find use-cases where `tmt-reboot -e` is used, at least in sourcegraph. Any other reference to check that this is os indeed safe to do?

Reply:
@LecrisUT unfortunately there is a single project using this:

https://sourcebot.dno.corp.redhat.com/~/search?query=content%3A%22tmt-reboot%20-e%22

Reply:
in public it is only tmt:

https://sourcegraph.com/search?q=context:global+content:%22tmt-reboot+-e%22&patternType=keyword&sm=0

Reply:
sending over a patch proposal for the user, to be merged once this live

Reply:
If we have an impact check (btw there are a couple more https://sourcebot.dno.corp.redhat.com/~/search?query=content%3A%20tmt-reboot.*-e) than I am fine with it. The notes I would have:
- Maybe do the impact check again right before the release to catch any new occurances
- Be more specific in the release notes that we flipped what `tmt-reboot -e` does